server/src/test-data.js

```diff
@@ -64,11 +71,15 @@ export const loadTestData = () =>
               group,
               name: 'Wollongong VR',
               details: 'Weekly VR availability',
+              startTime: 0, // ongoing
+              endTime: 0, // ongoing
             }),
             Creators.schedule({
               group,
               name: 'Emergency Services Expo',
               details: '5 people needed for demonstrations on Sunday',
+              startTime: 1512860400, // Sunday 10/12/17 @ 10am AEST in GMT
```

Comment:
no this isn't the correct stamp to store in the DB.
```
$ TZ=AEST date -d @1512860400
Sat Dec  9 23:00:00 AEST 2017
```

If you want to find the correct stamp use this:
```
$ TZ=AEST date -d '2017-12-10 10:00' +%s
1512900000
```

you can confirm like this:
```
$ TZ=AEST date -d @1512900000
Sun Dec 10 10:00:00 AEST 2017
```

Reply:
I dont think your right on that one.

1512860400
Is equivalent to:
12/09/2017 @ 11:00pm (UTC)

Add 11 hrs to that and you get 13/09/2017 @ 10.00am

Both a random handful of website, and the availability app agree with me on this one.

![screenshot 2017-12-07 15 10 49](https://user-images.githubusercontent.com/5799222/33711525-74ea387e-db98-11e7-8eee-7cbaa29bbada.png)

Reply:
Yeah you're totally right - `TZ=AEST` didn't work I guess cause you're in DST.
```
$ TZ=Australia/Sydney date -d @1512860400
Sun Dec 10 10:00:00 AEDT 2017
```
Your latest comment said this:
> Add 11 hrs to that and you get 13/09/2017 @ 10.00am

But I think you mean:
> Add 11 hrs to that and you get 12/10/2017 @ 10.00am

unrelated: why are you using an american locale?

Reply:
Blame website for locale.